### PR TITLE
[luci/test] Revise comment of evalverify.sh

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise how to use comment of evalverify.sh.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
From https://github.com/Samsung/ONE/pull/8348#discussion_r793180249

File: compiler/luci-value-test/evalverify.sh [hunk 1]
```diff
@@ -4,7 +4,8 @@
 #
 # HOW TO USE
 #
-# ./evalverify.sh <path/to/bin_dir> <path/to/work_dir> <path/to/venv_dir> <TEST 1> <TEST 2> ...
+# ./evalverify.sh <path/to/bin_dir> <path/to/work_dir> <path/to/venv_dir> <path/to/eval_driver> \
+#                 <TEST 1> <TEST 2> ...
 # bin_dir  : build directory of luci-value-test (ex: build/compiler/luci-value-test)
 # work_dir : artifacts directoy where test materials exist
 # venv_dir : python virtual environment home directory
```
Comment (hyunsik-yoon) [inline compiler/luci-value-test/evalverify.sh:11]: (optional) Maybe some detailed explanation about `#eval_driver` might be needed?

**Review by hyunsik-yoon — APPROVED** [review]

**Review by mhs4670go — DISMISSED** [review]
LGTM

**Review by hyunsik-yoon — DISMISSED** [review]
LGTM, thanks.

File: compiler/luci-value-test/evalverify.sh [hunk 2]
```diff
@@ -4,10 +4,12 @@
...
+# eval_driver : luci_eval_driver path for evavaluation
```
Comment (seanshpark) [inline compiler/luci-value-test/evalverify.sh:12]: typo `evavaluation`... -_-;;;

**Review by YongseopKim — APPROVED** [review]
LGTM

**Review by hyunsik-yoon — APPROVED** [review]

**Review by mhs4670go — APPROVED** [review]

**Pull request merged**